### ISSUE TITLE
Experiencing an unexpected cessation of Anki functionality due to an add-on issue.

### ISSUE THREAD

**Issue by 7moody112** [fonol/anki-search-inside-add-card#371]
Hey there! I noticed that after using Anki with an add-on for a few minutes, the app suddenly closes when I try to copy and paste something. Do you happen to know of any solutions for this issue? Thanks in advance!

**Screenshots**

**Environment (please complete the following information):**
 - OS: Mac book air  M1 chip 
 - Anki version⁨ 2.1.64 (581f82c5)⁩

https://github.com/fonol/anki-search-inside-add-card/assets/145294651/fee69f9d-12f2-4690-bc6b-7a27bf0b2888

**Comment by ronaldokun** [comment]
Yes, as described in the add-on page [here](https://ankiweb.net/shared/info/1781298089). The supported Anki versions are:

    2.1.1 (Updated 2020-04-01)
    2.1.22-2.1.49 (Updated 2021-09-17)
    2.1.51-2.1.54+ (Updated 2022-08-15) 

Unfortunately for newer versions it doesn't work yet.
